examples/cc26xx/cc26xx-web-demo/cc26xx-web-demo.c

```diff
@@ -464,6 +476,14 @@ get_batmon_reading(void *data)
     }
   }
 
+  if(adc_dio23_reading.publish) {
```

Comment:
I'd either change the function name to something suggesting that the adc is handled, too or move this into a seperate handler function.